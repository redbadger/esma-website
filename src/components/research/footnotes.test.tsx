import React from "react";
import renderer from "react-test-renderer";
import Footnotes from './footnotes';

describe("Footnotes", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Footnotes>
          <ol>
            <li>Sample entry</li>
          </ol>
        </Footnotes>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
