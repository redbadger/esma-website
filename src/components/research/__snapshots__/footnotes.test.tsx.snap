// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footnotes renders correctly 1`] = `
<div
  className="css-1v40p0u-footnoteStyles"
  id="footnotes"
>
  <h3>
    Footnotes
  </h3>
  <ol>
    <li>
      Sample entry
    </li>
  </ol>
</div>
`;
